moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -571,6 +571,13 @@ TEST_F(OneRobot, FK)
   EXPECT_NEAR_TRACED(state.getGlobalLinkTransform("link_e").translation(), Eigen::Vector3d(2.8, 0.6, 0));
 }
 
+// The generateTestTraj function should return a trajectory of length 7
```

Comment:
There is no reason to clutter the global space with these variables. They are only used in one or two tests!

Reply:
@rhaschke In general I think it's better to have a single version of data stored and referenced multiple places rather than multiple versions that could get out of sync. I'm not sure I see the issue with adding variables to the global space since this file is only built for testing and I don't see how the variables could leak outside of scope. That said, I removed that commit and made a small change that I think we can all agree is best on all fronts.